src/clr-angular/data/datagrid/enums/column-changes.enum.ts

```diff
@@ -6,4 +6,7 @@
 
 export enum DatagridColumnChanges {
   WIDTH,
+  HIDDEN,
 }
+
+export const ALL_COLUMN_CHANGES: DatagridColumnChanges[] = [DatagridColumnChanges.WIDTH, DatagridColumnChanges.HIDDEN];
```

Comment:
I think we could iterate over the keys of the enum to build ALL_COLUMN_CHANGES, untested but something like

```
export const ALL_COLUMN_CHANGES: DatagridColumnChanges[] = Object.keys(DatagridColumnChanges).map(key => DatagridColumnChanges[key]);```

Reply:
Ah yes, but we still need to filter out the string number properties from the object version. So this one will work for us:

```
export const ALL_COLUMN_CHANGES: DatagridColumnChanges[] = Object.keys(DatagridColumnChanges)
  .map(key => DatagridColumnChanges[key])
  .filter(key => key && isNaN(key));
```